tests/test_models.py

```diff
@@ -483,3 +490,30 @@ class UserAdmin(ModelView, model=User):
         column_default_sort = [("name", True), ("id", False)]
 
     assert UserAdmin()._get_default_sort() == [("name", True), ("id", False)]
+
+
+@pytest.mark.asyncio
+async def test_get_model_objects(session) -> None:
+    batman = User(name="batman")
+    bruce = User(name="bruce wayne")
+    superman = User(name="superman")
+    session.add(batman)
+    session.add(bruce)
+    session.add(superman)
+    session.commit()
+    session.refresh(batman)
+    session.refresh(bruce)
+    session.refresh(superman)
```

Comment:
It can simply work with one instance to prove the test. No need for the extra LOC.

Reply:
see last changes